60replicas.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
```

Comment:
The number of replicas is configurable as far as I can see, so I gather this is not the best script name :)